code/nrf-connect/samples/ble/src/encoding.c

```diff
@@ -104,33 +104,36 @@ int prst_ble_encode_service_data(const prst_sensors_t* sensors,
   out[4] = temp_val & 0xff;
   out[5] = temp_val >> 8;
   // Humidity.
-  out[6] = 0x03;
-  // Value. Factor 0.01, over 100%.
-  uint16_t humi_val = 10000 * sensors->shtc3.rel_humi;
-  out[7] = humi_val & 0xff;
-  out[8] = humi_val >> 8;
+  out[6] = 0x2E;
+  // Value. Factor 1 over 100%.
+  uint8_t humi_val = 100*sensors->shtc3.rel_humi + 0.5f;
```

Comment:
Do we need the `0.5f` term here and below?

Reply:
One of the "code formatting check" tests is failing (you can see [it here](https://github.com/rbaron/b-parasite/actions/runs/8764620429/job/24054923338?pr=188)).

It seems to be due to format of the `*` operator (it requires a space as in `a * b`). And another one is due to a trailing whitespace.